portfolio/src/main/webapp/script.js

```diff
@@ -13,11 +13,27 @@
 // limitations under the License.
 
 function fetchfunction(){
-    fetch("/data").then(response => response.json()).then((json) => {
-        document.getElementById('testing').innerText = json;
+    fetch("/data").then(response => response.json()).then((tasks) => {
+        for (i=0; i<tasks.length; i++) {
+            if (tasks[i]["propertyMap"]["comment"] != null) {
+                let node = document.createElement("div");
+                node.id = tasks[i]["key"]["id"];
+                node.innerText = tasks[i]["propertyMap"]["comment"] + "\n -- Date posted: "  + tasks[i]["propertyMap"]["time"] + "\n -- " + tasks[i]["propertyMap"]["name"] + "\n ";
+
+                let nodeTwo = document.createElement("button");
+                nodeTwo.id = tasks[i]["key"]["id"];
```

Comment:
you have two nodes here sharing the same ID. You might want to consider using classname instead for things that share an identifier.